dds/DCPS/transport/framework/TransportRegistry.h

```diff
@@ -157,15 +167,30 @@ class OpenDDS_Dcps_Export TransportRegistry {
     OPENDDS_STRING transport_template_name;
     OPENDDS_STRING config_name;
     bool instantiate_per_participant;
-    OPENDDS_MAP(OPENDDS_STRING, OPENDDS_STRING) customizations;
-    OPENDDS_MAP(OPENDDS_STRING, OPENDDS_STRING) transport_info;
+    ValueMap customizations;
+    ValueMap transport_info;
   };
 
   OPENDDS_VECTOR(TransportTemplate) transport_templates_;
 
   bool get_transport_template_info(const ACE_TString& config_name, TransportTemplate& inst);
 
-  bool has_transport_template() const;
+  bool has_transport_templates() const;
+
+  struct TransportEntry
+  {
+    OPENDDS_STRING transport_name;
+    OPENDDS_STRING config_name;
```

Comment:
TString since we're comparing these with what comes out of the configuration heap?